src/helics/core/helics_definitions.hpp

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
Shouldn't src/helics/helics_enums.h be included in this file?

Reply:
yes it should,  must have just included that in every file that also used this.

Reply:
Ok. Seeing as how this is a core header it won't be used at the application api layer it shouldn't affect the interface but I do suggest this be added before we forget.

Reply:
I added it, and I think this should be used by the application API, it will be included with the files for the application_api,  I doubt it would be included in isolation but it definitely needed that header.

Reply:
Well I meant normally external tools will most likely just use helics_enums.h. I normally wouldn't recommend them having to include headers form the core.